### PR TITLE
Add tab keybinding in commit message

### PR DESCRIPTION
**Pull request by navarroaxel**

### CODE REVIEW

**Comment by navarroaxel** [comment]
I was looking for a multi-line commit message and I found the `tab` key binding in a closed ticket, so I think is really good add the `tab` action to the panel. This is my PR:
Screenshot_20200722_082209

I tried to add the description for all the `Ctrl` keys but is too large:
Screenshot_20200722_081905

Let me know if you have any ideas how to describe all the commit message actions.

File: pkg/i18n/dutch.go [hunk 1]
```diff
@@ -214,6 +214,9 @@ func addDutch(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "NoTrackingThisBranch",
 			Other: "deze branch wordt niet gevolgd",
+		}, &i18n.Message{
+			ID:    "CommitMessage",
+			Other: "{{.keyBindClose}}: Sluiten, {{.keyBindNewLine}}: new line, {{.keyBindConfirm}}: Bevestigen",
```
Comment (navarroaxel) [inline pkg/i18n/dutch.go:219]: is `neue textzeile` the translation for `new line`?

Comment (mjarkk) [reply pkg/i18n/dutch.go:219]: Nope: "Nieuwe lijn"
Can you also change: "Bevestigen" to "Bevestig"

Comment (navarroaxel) [reply pkg/i18n/dutch.go:219]: the `CloseConfirm` message use: `Bevestigen`. Should I change this too?

Comment (mjarkk) [reply pkg/i18n/dutch.go:219]: Yes if you want to :)

**Comment by jesseduffield** [comment]
LGTM. I think the other keybindings are implicit in that typically you would use those bindings to navigate words in a terminal anyway. Thanks for making this!

**Pull request merged**